.github/workflows/integration-tests.yml

```diff
@@ -95,3 +95,15 @@ jobs:
         with:
           name: test-logs
           path: ./integration-tests/logs
+
+  solana:
+    name: Solana Tests
+    uses: smartcontractkit/chainlink-solana/.github/workflows/e2e.yml@reuseWorkflow
+    with:
+      repo_name: smartcontractkit/chainlink-solana
+    secrets:
```

Comment:
Do we need to do anything here to ensure that the built Chainlink image is used, rather than just default?